test.sh

```diff
@@ -52,6 +52,19 @@ fi
 cd ..
 rm -rf $DIRNAME
 
+echo "Testing: kitura init"
```

Comment:
If we're running a full build here we can probably delete the `Testing: kitura init --skip-build` section above?